tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -484,6 +484,61 @@ public function data_provider_not_supported_webp_images() {
 		yield 'GIFT image' => array( TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/earth.gif' );
 	}
 
+	/**
+	 * Ensure jpg image is replaced if the request is for the frontend context.
+	 *
+	 * @group webp_uploads_update_image_references
+	 *
+	 * @test
+	 */
+	public function it_should_not_replace_image_references_when_request_is_for_frontend_context() {
```

Comment:
I think the "not" here is a typo? We can also shorten this a bit.
```suggestion
	public function it_should_replace_image_references_when_in_frontend_context() {
```